docs/UserGuide.md

```diff
@@ -454,7 +483,7 @@ Launches GitHub repository of the specified person on the browser from the addre
 <box type="definition" icon=":fa-solid-book:" light>
 
 <md>**Examples:**</md>
-
+* `git name/James` launches the GitHub repository of the person named `James`, using the GitHub username assigned to `James`. It uses the shortcut command.
```

Comment:
I feel it would be better to put the shortcut command and parameter in the same example instead of splitting it similar to lines 556 to 558.

So something like: 
`github name/Harry` ...
`git n/Harry`...

Reply:
Good point! Less confusion for the reader too!